core/src/services/dropbox/core.rs

```diff
@@ -111,6 +111,23 @@ impl DropboxCore {
         self.client.send(request).await
     }
 
+    pub async fn dropbox_get_stat(&self, path: &str) -> Result<Response<IncomingAsyncBody>> {
```

Comment:
Please naming the function in the same way as it's API, like: `dropbox_get_metadata`